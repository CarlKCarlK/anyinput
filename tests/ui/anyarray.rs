use anyinput::anyinput;

#[anyinput]
fn empty_array(array: AnyArray) {
    println!(array[0]);
}

fn main() {}
