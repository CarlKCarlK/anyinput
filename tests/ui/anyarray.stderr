error: custom attribute panicked
 --> tests/ui/anyarray.rs:3:1
  |
3 | #[anyinput]
  | ^^^^^^^^^^^
  |
  = help: message: AnyArray expects a generic parameter, for example, AnyArray<usize> or AnyArray<AnyString>.
